src/server/controller/cycleData/updateDependencies/updateCalculationDependencies/context.ts

```diff
@@ -0,0 +1,57 @@
+import { CountryIso } from 'meta/area'
+import { Assessment, Cycle, VariableCache } from 'meta/assessment'
+import { NodeUpdate, NodeUpdates } from 'meta/data'
+
+type ConstructorProps = {
+  assessment: Assessment
+  cycle: Cycle
+  countryIso: CountryIso
+}
+
+export class Context {
+  private readonly _assessment: Assessment
```

Comment:
would it make sense to use private class fields?
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Private_class_fields